src/main/scala/com/rbmhtechnology/eventuate/sandbox/ReplicationProtocol.scala

```diff
@@ -0,0 +1,19 @@
+package com.rbmhtechnology.eventuate.sandbox
+
+import akka.actor.ActorRef
+
+import scala.collection.immutable.Seq
+
+object ReplicationProtocol {
+  case class GetReplicationSourceLogs(logNames: Set[String])
+  case class GetReplicationSourceLogsSuccess(endpointId: String, sourceLogs: Map[String, ActorRef])
+
+  case class GetReplicationProgressAndVersionVector(sourceLogId: String)
+  case class GetReplicationProgressAndVersionVectorSuccess(progress: Long, targetVersionVector: VectorTime)
+
+  case class ReplicationRead(fromSequenceNr: Long, num: Int, targetLogId: String, targetVersionVector: VectorTime)
```

Comment:
I'd say no because the target i.e. consumer should decide how much load (= events per batch) it can process.